geode-core/src/integrationTest/java/org/apache/geode/net/SSLSocketParameterExtensionIntegrationTest.java

```diff
@@ -216,17 +217,18 @@ public void uncaughtException(final Thread thread, final Throwable throwable) {
   }
 
   public static class MySSLParameterExtension implements SSLParameterExtension {
-    DistributionConfig config;
+    Map<SSLParameterExtensionTypes, Object> config;
```

Comment:
While this addresses the compile time discovery of the value it doesn't address its typing or issues with runtime existence of the value in the map like a concrete object would. The `Map` contract on `get` says it will return `null` if the key does not exist. At runtime I must check for `null` results and respond appropriately. A non-null value must then be cast to the correct type at runtime, possibly resulting an a runtime exception. What initializes this `Map` with default values? If we address all those issue we now have a class.

Reply:
@pivotal-jbarrett I have submitted changes to address this. Please re-review and provide further feedback.